### PR TITLE
DEVO-204:Update develop pipeline to run full develop pipeline nightly

### PR DESCRIPTION
**Pull request by yafeldman**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -382,7 +382,6 @@ pipeline{
 	agent none;
 	options {
   	checkoutToSubdirectory 'data-hub'
-  	skipStagesAfterUnstable()
```
Comment (SameeraPriyathamTadikonda) [inline Jenkinsfile:385]: If we remove this. PR's will be merged even if tests fail.

**Review by SameeraPriyathamTadikonda — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (yafeldman) [reply Jenkinsfile:385]: if any of the tests fail then stage('tests') will be unstable. Then post section in lines 549-554 will abort entire build with failure.

Comment (SameeraPriyathamTadikonda) [reply Jenkinsfile:385]: got it. Thanks for confirming.

**Review by SameeraPriyathamTadikonda — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (yafeldman) [reply Jenkinsfile:385]: > If we remove this. PR's will be merged even if tests fail.
if any of the tests fail then stage('tests') will be unstable. Then post section in lines 549-554 will abort entire build with failure.

**Review by kkanthet — APPROVED** [review]

**Pull request merged**